### PR TITLE
Android dropdown req page

### PR DESCRIPTION
**Pull request by heartjigi**
Fix dropdown on request page,
It has problem after adding custom keyboard.
and It is fixed now!

and fix some font Style for new text usage

### CODE REVIEW

File: android/app/src/main/java/com/yoda/MainApplication.java [hunk 1]
```diff
@@ -3,6 +3,10 @@
 import com.evollu.react.fcm.FIRMessagingPackage;
 import com.facebook.appevents.AppEventsLogger;
 import android.app.Application;
+import android.content.Context;
+import android.support.multidex.MultiDex;
```
Comment (SarahhhC) [inline android/app/src/main/java/com/yoda/MainApplication.java:7]: What is MultiDex?

File: android/app/src/main/java/com/yoda/MainApplication.java [hunk 2]
```diff
@@ -57,4 +61,10 @@ public void onCreate() {
...
         AppEventsLogger.activateApp(this);
     }
-}
+
+    @Override
+    protected void attachBaseContext(Context base) {
+        super.attachBaseContext(base);
+        MultiDex.install(this);
+    }
+}
```
Comment (SarahhhC) [inline android/app/src/main/java/com/yoda/MainApplication.java]: Add a new line :)

File: android/app/src/main/java/com/yoda/MainApplication.java (hunk 1 above)
Comment (heartjigi) [reply android/app/src/main/java/com/yoda/MainApplication.java:7]: for  android version under 5

**Comment by SarahhhC** [comment]
Good LGTM!! 👍

**Pull request merged**